spec/searchworks_format_spec.rb

```diff
@@ -588,6 +588,17 @@
       @smods_rec.from_str(m)
       expect(@smods_rec.sw_genre).to eq ['Thesis/Dissertation']
     end
+    it "Technical Report: typeOfResource 'text', genre 'technical report'" do
+      m = "<mods #{@ns_decl}><genre authority=\"marcgt\">technical report</genre><typeOfResource>text</typeOfResource></mods>"
+      @smods_rec.from_str(m)
+      expect(@smods_rec.sw_genre).to eq ['Technical report']
+      m = "<mods #{@ns_decl}><genre authority=\"marcgt\">Technical report</genre><typeOfResource>text</typeOfResource></mods>"
+      @smods_rec.from_str(m)
+      expect(@smods_rec.sw_genre).to eq ['Technical report']
+      m = "<mods #{@ns_decl}><genre authority=\"marcgt\">Technical Report</genre><typeOfResource>text</typeOfResource></mods>"
+      @smods_rec.from_str(m)
+      expect(@smods_rec.sw_genre).to eq ['Technical report']
+    end
```

Comment:
see other PR re:  possibly adding test for when there is no authority attrib in genre field?